packages/web-styles/src/mixins/_forms.scss

```diff
@@ -85,31 +85,6 @@
       }
     }
   }
-
-  .custom-control-input {
```

Comment:
Why was this removed?

Reply:
I removed this because it was dead code: CWF no longer offers `.custom-control-input`'s in v5